### PR TITLE
Add code of conduct section and links to Summit events

### PR DESCRIPTION
**Pull request by robinske**
I feel better about encouraging people to apply for the CFP/attend the event when things like this are made explicit.

Addresses https://github.com/typelevel/typelevel.github.com/issues/139

### CODE REVIEW

File: _events/2017-03-23-summit-nyc.md [hunk 1]
```diff
@@ -72,3 +72,7 @@ For more information about the venue and accommodation, check out the [NE Scala
 If you would like to talk to us about sponsorship options, please get in touch with us:
 
 <a class="btn large" href="mailto:info@typelevel.org">Become a sponsor</a>
+
+## Code of Conduct
+
+The Typelevel Summit is a community conference and we strive to make it an inclusive and fulfilling event for all participants. All attendees, speakers, and organizers must abide by the [Typelevel Code of Conduct](http://typelevel.org/conduct.html)
```
Comment (tpolecat) [inline _events/2017-03-23-summit-nyc.md]: Period at end of sentence? (both files)

**Comment by robinske** [comment]
@tpolecat at the end of the `About the Summit` section? That works for me!

**Comment by tpolecat** [comment]
Maybe see what @larsrh thinks.

**Comment by larsrh** [comment]
@tpolecat Yes, good idea. @robinske, could you move the paragraph please?

**Comment by robinske** [comment]
@larsrh @tpolecat updated!

**Comment by larsrh** [comment]
Thank you!

**Pull request merged**